core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -384,19 +385,19 @@ class HttpGoogleServicesDAO(
     }
   }
 
-  override def getBucketUsage(googleProject: GoogleProjectId, bucketName: String, maxResults: Option[Long]): Future[BigInt] = {
+  override def getBucketUsage(googleProject: GoogleProjectId, bucketName: String, maxResults: Option[Long]): Future[BucketUsageResponse] = {
     implicit val service = GoogleInstrumentedService.Storage
 
-    def usageFromLogObject(o: StorageObject): Future[BigInt] = {
+    def usageFromLogObject(o: StorageObject): Future[BucketUsageResponse] = {
       streamObject(o.getBucket, o.getName) { inputStream =>
         val content = Source.fromInputStream(inputStream).mkString
         val byteHours = BigInt(content.split('\n')(1).split(',')(1).replace("\"", ""))
         // convert byte/hours to byte/days to better match the billing unit of GB/days
-        byteHours / 24
+        BucketUsageResponse(byteHours / 24, new DateTime(o.getUpdated.getValue))
```

Comment:
❓ Consider adding a null check here, since the [javadoc](https://developers.google.com/resources/api-libraries/documentation/storage/v1/java/latest/com/google/api/services/storage/model/StorageObject.html) states this method may return null

Reply:
Good call! I'll add that in